### PR TITLE
Fix some golint failures for pkg/controller

### PR DESCRIPTION
**Pull request by pmcalpine**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix golint failures on :

* pkg/controller/bootstrap
* pkg/controller/cronjob

**Which issue(s) this PR fixes**:

Refs #68026

**Special notes for your reviewer**:

This is my first k8s PR. Thanks for your grace as I learn how to navigate the process.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/cronjob/utils.go [hunk 1]
```diff
@@ -183,6 +183,7 @@ func getFinishedStatus(j *batchv1.Job) (bool, batchv1.JobConditionType) {
 	return false, ""
 }
 
+// IsJobFinished returns whether or not a job has finished.
```
Comment (cheftako) [inline pkg/controller/cronjob/utils.go]: Might be worth mentioning that its based on if its been marked as finished irrespective of it it was successfully.

**Review by cheftako — COMMENTED** [review]
/lgtm

**Review by janetkuo — COMMENTED** [review]
`pull-kubernetes-verify` test failure seems to be related to this change: 
```
cmd/kube-controller-manager/app/batch.go:49:14: undefined: cronjob.NewCronJobController
```

**Review by cheftako — COMMENTED** [review]
/lgtm

**Comment by pmcalpine** [comment]
@janetkuo could you PTAL? I'm not sure how to advance past the tide presubmit. Do I need to rebase/re-push?

**Review by janetkuo — APPROVED** [review]

**Comment by deads2k** [comment]
> /lgtm

based on the title and Walter's lgtm

/approve

**Comment by soltysh** [comment]
Why `cronjob_controller.go` was renamed to `controller.go`, there's a valid reason we keep the file names as they were, which is the logs contain only the file name, which when we start renaming all controller files to `controller.go` will make the logs completely unreadable, see https://github.com/kubernetes/kubernetes/pull/37208 for historical reasons why. I've opened https://github.com/kubernetes/kubernetes/pull/77060 to bring back the old name, please don't change those in the future.

**Comment by pmcalpine** [comment]
> Why `cronjob_controller.go` was renamed to `controller.go`, there's a valid reason we keep the file names as they were, which is the logs contain only the file name, which when we start renaming all controller files to `controller.go` will make the logs completely unreadable, see #37208 for historical reasons why. I've opened #77060 to bring back the old name, please don't change those in the future.

I've replied on #77060.

**Pull request merged**